lab_3_ann_retriever/main.py

```diff
@@ -207,6 +305,17 @@ def load(self, file_path: str) -> bool:
 
         In case of corrupt input arguments, False is returned.
         """
+        with open(file_path, 'r', encoding='utf-8') as file:
+            state = load(file)
+            if not 'idf_values' in state.keys() or not 'vocabulary' in state.keys()\
```

Comment:
check `'idf_values' in state.keys()` the same as `'idf_values' in state`

Reply:
`not 'idf_values' in state.keys()` - how doy you interpreter it? Everything except `'idf_values'` is in `state`, right? `not` is in wrong palce.

Reply:
changed